spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+require 'pry'
+
+RSpec.feature 'Creating Comments' do
+
+  before do
+    @foo = User.create(email: "foo@bar.com", password: "password", password_confirmation: "password")
```

Comment:
Metrics/LineLength: Line is too long. [101/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)